pkg/core/storage/boltdb_store.go

```diff
@@ -0,0 +1,129 @@
+package storage
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/CityOfZion/neo-go/config"
+	"github.com/boltdb/bolt"
+	"github.com/syndtr/goleveldb/leveldb/util"
```

Comment:
It's strange to see BoltDB code using LevelDB, but if it's really useful (and looks like it is) then whatever...